sys/include/net/ipv6/addr.h

```diff
@@ -113,6 +110,11 @@ typedef union {
                                                0x00, 0x00, 0x00, 0x00, \
                                                0x00, 0x00, 0x00, 0x00 }}
 
+/**
+ * @brief   Length of the link local prefix
+ */
+#define IPV6_ADDR_LINK_LOCAL_PREFIX_LEN     (8U)
```

Comment:
When it comes to IP addresses, prefix length is usually expressed in bits, so using bytes here in this public definition might be confusing unless its indicated in the name.
```suggestion
#define IPV6_ADDR_LINK_LOCAL_PREFIX_BYTES   (8U)
```